### DISCUSSION TITLE
"The universality of the HI to dark matter halo mass ratio for isolated disk galaxies"

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#20] [Astrophysics and Physics]
Talk by Dr. Marie Korsaga  (ObAS and UJKZ) given at the Instituto de Astrofísica de Canarias on May 25th, 2023.
Submitted paper, Korsaga *et al.*

On [YouTube https://youtu.be/-lly6HXpvh8](https://youtu.be/-lly6HXpvh8) (*Unfortunately the audio is of poor quality*)

Of interest: Why is the hydrogen content of a galaxy related to its dark matter halo content with simple monotonous relationships?

> One of the most active areas of research of the last decade is undoubtedly the study of the effects of baryons on the observed dynamics of galaxies. This particularly led to the establishment of some fundamental scaling relations, which characterise the dependence of the abundance of baryons on the properties of galaxies' dark matter haloes. Among these fundamental relations, the stellar–to-halo mass relation appears to be one of the most investigated.  In this talk, I will present the less commonly explored **neutral hydrogen-to-halo mass relation** constructed using high-quality extended HI rotation curves of isolated rotationally-supported disk galaxies, selected from the SPARC and LITTLE THINGS databases. I will discuss how we constrain the dark matter halo of these galaxies using a Navarro-Frenk-White cuspy density profile and a semi-analytic Dekel-Zhao density profile and how we investigate the scaling relations between baryons and halo parameters.

**Comment by budrap00** [comment]
Sounds to me like they're just trying to fit their Sacred DM Halo to the Radial Acceleration Relation, no?

**Reply by RedshiftDrift** [reply]
Yes, but there is something else.  They find a simple correspondence between the amount of DM and the amount of atomic hydrogen.
This is interesting because since DM doesn't exist, HI may have a role in the correct explanation for what is thought to be dark matter.
